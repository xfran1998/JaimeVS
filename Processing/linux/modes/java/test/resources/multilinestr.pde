String testOldStyle = "line1\"\nline 2 \"\"\nline  3";
String testMultiline = """
line4 "
line 5 ""
line  6
line   7""";

println(testOldStyle);
println(testMultiline);
