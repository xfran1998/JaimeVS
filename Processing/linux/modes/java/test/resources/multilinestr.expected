import processing.core.*;
import processing.data.*;
import processing.event.*;
import processing.opengl.*;

import java.util.HashMap;
import java.util.ArrayList;
import java.io.File;
import java.io.BufferedReader;
import java.io.PrintWriter;
import java.io.InputStream;
import java.io.OutputStream;
import java.io.IOException;

public class multilinestr extends PApplet {

    public void setup() {
String testOldStyle = "line1\"\nline 2 \"\"\nline  3";
String testMultiline = "\nline4 \"\nline 5 \"\"\nline  6\nline   7";

println(testOldStyle);
println(testMultiline);

        noLoop();
    }

    static public void main(String[] passedArgs) {
        String[] appletArgs = new String[] { "multilinestr" };
        if (passedArgs != null) {
            PApplet.main(concat(appletArgs, passedArgs));
        } else {
            PApplet.main(appletArgs);
        }
    }
}
